cmd/ctr/shim.go

```diff
@@ -133,11 +137,15 @@ var shimStartCommand = cli.Command{
 	Name:  "start",
 	Usage: "start a container with a shim",
 	Action: func(context *cli.Context) error {
-		service, err := getShimService()
+		var (
```

Comment:
I enjoy either the syntax for var(), or the syntax := in Golang. However, var() is clear to read , and is used widely in cmd/ctr.
So I suggest that it make sense to use var()